sdl2-sys/build.rs

```diff
@@ -555,15 +555,11 @@ fn main() {
 
     #[cfg(feature = "bindgen")]
     {
-        let include_paths: Vec<String>;
-        #[cfg(feature = "bundled")]
-        {
-            include_paths = vec![sdl2_includes];
-        }
-        #[cfg(not(feature = "bundled"))]
-        {
-            include_paths = compute_include_paths(sdl2_includes)
-        }
+        let include_paths: Vec<String> = if cfg!(feature = "bundled") {
+            vec![sdl2_includes]
+        } else {
+            compute_include_paths(sdl2_includes)
```

Comment:
This fails to compile with `bindgen` and `bundled` features enabled, so please put the call to `compute_include_paths` back behind `#[cfg(not(feature = "bundled"))]`.

I get that the formatting could be better, how about this instead?
```rust
#[cfg(feature = "bundled")]
let include_paths = vec![sdl2_includes];

#[cfg(not(feature = "bundled"))]
let include_paths = compute_include_paths(sdl2_includes);
```